tests/io/test_parquet_roundtrip.py

```diff
@@ -1,159 +1,17 @@
 from __future__ import annotations
 
-import datetime
-import decimal
 import random
+from pathlib import Path
 
-import numpy as np
 import pyarrow as pa
 import pyarrow.parquet as papq
 import pytest
 
 import daft
-from daft import DataType, Series, TimeUnit
```

Comment:
Ah ok, I see I did a bad job separating out the refactoring changes into the other branch. I need to fix these before merging.